### PR TITLE
Split features tests into multiple files

### PR DESCRIPTION
**Pull request by axelboc**
Even though we're still always rendering the whole app, I think splitting the tests based on the component being tested makes sense. `App.test.tsx` file was becoming too unwieldy.

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -52,8 +52,6 @@ module.exports = createConfig({
     {
       files: jestFiles,
       rules: {
-        'jest/require-top-level-describe': 'off', // until App.test.tsx is split into multiple files
-
```
Comment (axelboc) [inline .eslintrc.js:56]: I've added a `describe` block in each test file.

File: src/h5web/explorer/Explorer.test.tsx [hunk 2]
```diff
@@ -0,0 +1,86 @@
+import { fireEvent, screen } from '@testing-library/react';
+import { DOMAIN, renderApp } from '../test-utils';
+
+describe('Explorer', () => {
+  test('select root group by default', async () => {
+    renderApp();
```
Comment (loichuder) [inline src/h5web/explorer/Explorer.test.tsx:6]: Is the app created without any mock data here ?

Comment (axelboc) [reply src/h5web/explorer/Explorer.test.tsx:6]: Nothing's changed, it renders the `App` with the `MockProvider`, so it renders exactly the same data as the `/mock` route.

Comment (loichuder) [reply src/h5web/explorer/Explorer.test.tsx:6]: Read until the end: it has mock data.

Then, perhaps the check on `Nothing to visualize` is a bit tricky as the root node might have some visualisations with NeXuS.

Comment (axelboc) [reply src/h5web/explorer/Explorer.test.tsx:6]: Hmm good point... Seems to be passing when it shouldn't...

**Review by loichuder — APPROVED** [review]

**Pull request merged**